### PR TITLE
Added 4 box-shadow-overlapping-* tests

### PR DESCRIPTION
**Pull request by TalbotG**
box-shadow-overlapping-001.html
box-shadow-overlapping-002.html
box-shadow-overlapping-003.html
box-shadow-overlapping-004.html
reference/box-shadow-overlapping-001-ref.html
reference/box-shadow-overlapping-003-ref.html

### CODE REVIEW

File: css/css-backgrounds/box-shadow-overlapping-002.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  <title>CSS Background and Borders Test: box-shadow and overlapping of adjacent text</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#shadow-layers" >
+  <link rel="match" href="reference/box-shadow-overlapping-001-ref.html">
+  <link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+
+  <meta content="" name="flags">
+  <meta content="Box shadows do not affect layout. Box shadows take up no space. Box shadows do not affect or influence normal flow of boxes. Therefore, box shadows can 'collide' with other boxes and can overlap other boxes (inline box or line box) or be overlapped by other boxes. In this test, the text ('PNG') follows a right outer box shadow and therefore overlaps such shadow. The box shadow, even though it is overlapped by the text ('PNG'), nevertheless overlaps the background color (yellow) of the block element." name="assert">
```
Comment (fantasai) [inline css/css-backgrounds/box-shadow-overlapping-002.html:13]: Your test is not checking "nevertheless overlaps the background color (yellow) of the block element".

File: css/css-backgrounds/box-shadow-overlapping-002.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  <style>
+  div
+    {
+      background-color: yellow;
+      color: green;
+      float: left;
+      font-family: Ahem;
+      font-size: 100px;
+      line-height: 1;
```
Comment (fantasai) [inline css/css-backgrounds/box-shadow-overlapping-002.html:23]: font: 100px/1 Ahem;

There's no reason to be wordy here.

**Review by fantasai — CHANGES_REQUESTED** [review]
Not bad. A few minor comments.

**Comment by fantasai** [comment]
For future reference: I think this is another example of a series of tests that would be more efficient to run as a single file, but because the understandability separately or together is equal, I will not ask you to make the change to combine them. :)

File: css/css-backgrounds/box-shadow-overlapping-002.html (hunk 2 above)
Comment (TalbotG) [reply css/css-backgrounds/box-shadow-overlapping-002.html:23]: There is a reason why. If I need to see where the P, E, D, P, N, G glyphs are, then I can toggle on/off the font-family in the web browser inspection tool. I would not be able to do this easily and quickly if I had used a shorthand form. Use of longhand form for font related properties is only for convenience.

File: css/css-backgrounds/box-shadow-overlapping-002.html (hunk 1 above)
Comment (TalbotG) [reply css/css-backgrounds/box-shadow-overlapping-002.html:13]: Correct. box-shadow-overlapping-002.html does not check the "nevertheless overlaps the background color (yellow) of the block element" statement in the assert text. Another test, specifically designed to check this, could be created. Here, in box-shadow-overlapping-002.html, I will just remove this "nevertheless overlaps the background color (yellow) of the block element".

**Comment by TalbotG** [comment]
I need to close this PR. I will create a new PR in a few min. I will include your comments and mine.

**Pull request closed without merging**